packages/media/src/formatTimeFromLength.js

```diff
@@ -0,0 +1,40 @@
+/*
+ * Copyright 2021 Google LLC
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     https://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+/**
+ * Internal dependencies
+ */
+import formatDuration from './formatDuration';
+
+/**
+ * Format time to be human readable
+ *
+ * @param {number} length Length in seconds.
+ * @return string
+ */
+const formatTimeFromLength = ( length ) => {
```

Comment:
Can we add tests?

Also, does this make `formatMsToHMS` obsolete? Or how do they relate?

Reply:
Looks like the difference is that this one takes in the original value in seconds. These could still probably be unified, it could always take milliseconds in.

Reply:
So perhaps we could just adjust `formatMsToHMS` to skip the hours when irrelevant and just use it with `length * 1000` in `getVideoLength` and then we wouldn't need a second formatter?

Reply:
@spacedmonkey I'll make that change directly in the origin PR, hope you don't mind.